index.js

```diff
@@ -6,5 +6,201 @@ function closeNav() {
+body.appendChild(div1);
+
+for(var i = 0; i < cardObjects.length; i++) {
+  cardObjects[i].cardbutton = cardBtn[i];
+  if(cardObjects[i].cardbutton.classList.contains('myBtn1')) {
+    cardObjects[i]=card1;
+    cardObjects[i].cardbutton = cardBtn[0];
+    var btn = cardObjects[i].cardbutton;
+    btn.onclick = function () {
+    var modal = document.getElementById("myModal");
+    modal.style.display = "block";
+    h2.innerHTML = cardObjects[i].title;
+    div2.insertBefore(cardObjects[i].Image, div4);
+    div2.insertBefore(cardObjects[i].info, cardObjects[i].Image);
+    div5.insertBefore(cardObjects[i].technologies, div6);
+    };
+  } 
+  if(cardObjects[i].cardbutton.classList.contains("myBtn2")) {
+    cardObjects[i].cardbutton = cardBtn[1];
+    var btn = cardObjects[i].cardbutton;
+    btn.onclick = function () {
+    var modal = document.getElementById("myModal");
+    modal.style.display = "block";
+    h2.innerHTML = cardObjects[i].title;
+    div2.insertBefore(cardObjects[i].Image, div4);
+    div2.insertBefore(cardObjects[i].info, cardObjects[i].Image);
+    div5.insertBefore(cardObjects[i].technologies, div6);
+    };
+  } 
+  if(cardObjects[i].cardbutton.classList.contains("myBtn3")) {
+    cardObjects[i].cardbutton = cardBtn[2];
+    var btn = cardObjects[i].cardbutton;
+    btn.onclick = function () {
+    var modal = document.getElementById("myModal");
+    modal.style.display = "block";
+    h2.innerHTML = cardObjects[i].title;
+    div2.insertBefore(cardObjects[i].Image, div4);
+    div2.insertBefore(cardObjects[i].info, cardObjects[i].Image);
+    div5.insertBefore(cardObjects[i].technologies, div6);
+    };
+  }
+  else if (cardObjects[i].cardbutton.classList.contains("myBtn4")) {
+    cardObjects[i].cardbutton = cardBtn[3];
+    var btn = cardObjects[i].cardbutton;
+    btn.onclick = function () {
+    var modal = document.getElementById("myModal");
+    modal.style.display = "block";
+    h2.innerHTML = cardObjects[i].title;
+    div2.insertBefore(cardObjects[i].Image, div4);
+    div2.insertBefore(cardObjects[i].info, cardObjects[i].Image);
+    div5.insertBefore(cardObjects[i].technologies, div6);
+  };
+}
+}
+
+//closing button for modal
+
+var modal = document.getElementById("myModal");
+var span = document.getElementsByClassName("close")[0];
+span.onclick = function () {
+  modal.style.display = "none";
+};
+window.onclick = function (event) {
+ if (event.target == modal) {
+ modal.style.display = "none";
+}
+};
+
+// Validation of form
+
+const form = document.getElementById('myForm');
+const email = document.getElementById('user_email');
+const errorMessage = document.getElementById('emsg');
+form.addEventListener('submit', (x) => {
+  if (email.value.toLowerCase() !== email.value) {
+    x.preventDefault();
+    errorMessage.textContent = 'Use only lowercase letters for your e-mail!';
+    errorMessage.style.color = "red";
+    errorMessage.style.backgroundColor = "yellow";
+  }
+});
```

Comment:
- [ ] Error message has to fits the design(
font, colors, images, tests, margins)  please fix that

Reply:
Dear @DeliceLydia,

It is fixed, please check

MR

Elmar